wolox-books/src/scss/application.scss

```diff
@@ -1 +1 @@
-/* You can add global styles to this file, and also import other style files */
+@import '../../layout/scss/equalizer';
```

Comment:
Actually, we have our own [equalizer](https://github.com/wolox/equalizer) so you can import it as a dependency and not worrying about having to add it manually 😄